org.upscayl.app.yml

```diff
@@ -0,0 +1,70 @@
+id: org.upscayl.app
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: upscayl-run
+separate-locales: false
+
+finish-args:
+  - --allow=devel
+  - --allow=multiarch
```

Comment:
Why does your app need these two?

Reply:
@barthalion Thanks for the review.

> Why does your app need these two?

I thought it would be better to include these for compatibility. I guess I'll remove them then. Thanks!